cmd/argocd/commands/app.go

```diff
@@ -1603,15 +1606,16 @@ func groupResourceStates(app *argoappv1.Application, selectedResources []argoapp
 	return resStates
 }
 
-func checkResourceStatus(watchSync bool, watchHealth bool, watchOperation bool, watchSuspended bool, healthStatus string, syncStatus string, operationStatus *argoappv1.Operation) bool {
+func checkResourceStatus(watchSync bool, watchHealth bool, watchOperation bool, watchSuspended bool, watchDegraded bool, healthStatus string, syncStatus string, operationStatus *argoappv1.Operation) bool {
```

Comment:
Since we are checking multiple conditions in `checkResourceStatus()`  we could add a include unit test in `app_test.go`

Reply:
@chetan-rns Agreed, I just added and pushed test coverage for `checkResourceStatus()`